codalab/model/bundle_model.py

```diff
@@ -30,6 +30,8 @@
     bundle as cl_bundle,
     bundle_dependency as cl_bundle_dependency,
     bundle_metadata as cl_bundle_metadata,
```

Comment:
Can you please update the PR description to describe what issue it fixes, what it does, etc.?